service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1873,6 +1914,71 @@ components:
             of the object.
           type: integer
 
+    CloneControlledGcpGcsBucketRequest:
+      description: >-
+        Values to assign to the cloned bucket for a given source bucket. Leave
+        name and description null to take the source object's values.
+        TODO: should we allow setting the destination bucket's cloning
+        instructions as well?
+      type: object
+      required: [destinationWorkspaceId, jobControl]
+      properties:
+        destinationWorkspaceId:
+          description: ID of workspace to be cloned into
+          type: string
+          format: uuid
+        cloningInstructions:
+          $ref: "#/components/schemas/CloningInstructionsEnum"
+        name:
+          $ref: "#/components/schemas/Name"
+        description:
+          type: string
+        bucketName:
+          description: >-
+            Name of created bucket (not the resource name). This name must
+            be globally unique, so it can't be a copy of the original bucket name.
+            It also can't be a simple formula like copy-of-bucket-name, as that would
+            fail on subsequent clone operations with bucket-name as the source bucket.
+            If null, a random bucket name will be created.
+          type: string
+        location:
+          description: >-
+            A valid bucket location per https://cloud.google.com/storage/docs/locations.
+            If null, will use source bucket's location.
+          type: string
+        jobControl:
+          $ref: '#/components/schemas/JobControl'
+    ClonedControlledGcpGcsBucket:
+      description: >-
+        Response object for clone of a GCS Bucket. Destination workspace and
+        resource IDs are available in the bucket's metadata object. TODO: generalize so there's a
+        single response type for all resource clones?
+      type: object
+      properties:
+        effectiveCloningInstructions:
+          $ref: '#/components/schemas/CloningInstructionsEnum'
+        sourceWorkspaceId:
+          description: ID of the workspace of the source resource for this clone
+          type: string
+          format: uuid
+        sourceResourceId:
+          description: ID of the source resource
+          type: string
+          format: uuid
+        bucket:
+          $ref: "#/components/schemas/CreatedControlledGcpGcsBucket"
+    CloneControlledGcpGcsBucketResult:
```

Comment:
I think we need to add something to this result about what we did.
Maybe that is returning the effective clone operation. That would allow us to express "did nothing" in this response.